src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -93,6 +93,9 @@ const activityFeedInlineError: Errors = {
         content: messages.activityFeedItemApiError
     }
 };
+
+const RESIN_INFOPANE_TAG = 'infopane';
```

Comment:
src/interactionTargets.js

Reply:
also call it 'sidebar' to be consistent

Reply:
`infopane` was what is requested as the component name, to be consistent with Webapp. Leaving as is